docs/tutorials/dt_thermal_emission.ipynb

```diff
@@ -0,0 +1,376 @@
v7+/1fN68eJFqqur6e62+DWh0WhISUkhPT1dFumanZ2lvr6e+vp6m+6Njo6OpKenk5qayvDwsNVG3O7u7sTFxRETEyPr0bYccysAs0hbflzNeHt7ExYWRnh4OD4+PpLrfGxsjKKiIk6cuBHSdnZ2JiEhgdjYWHQ6HX19fZSWlt50lG49qZRarRaNRiOrcTNH1JaLN3t7e5KTk0lJSZEZuVhDuV8fTt7Yf4Dqia28cTmSWaO9bLxGBZ9LgqcTwM5GZvtMSQNj33mBhZ4zkuF2Idvw+uU30caE34lNV1gFpQ+bwtsGURRHANmb6Z2wS1ZQuFlEEUrPwA+r4ZR1s0CcHeDTifDRXdZdwFZfvsjVAxVc+t6/sXhWbl9ut90fty9+GP278hCUfmq3hCiKdHR0cPjw4VV7qc3Ozkps8s1RlJVibcuWLeTn5xMQEGBJfVwp1HQ6HQkJCUxOTlJaWmp5oRcEgV27dpGdnY2rqyuzs7NUVlbS2NgoaT7t5uZGfn4+ERERVp+NAwMDVFdXS0xIBEEgLi6O7OxsWW3b7OwsdXV1NDQ02BRqer2egoICoqOj6e/v5w9/+IPMsTIiIsJmDdxyJicnaWlpoaWlRdZUWxAE/P39LSJttRowT09PPvCBD3D69GkOHz7MyMiI5ZiWl5cTHh5OQkICTz31FCMjIxw8eHDVWkNr4sz892rCzXzdrHSVXCnU5ufnmZ+fp6qqioaGBot753qEm8LDhUqAdEM/MfoRGh338FqPdPyi0eQsfPgU/GwPhFtpmeiUl4xfdjwT//U64z/4FeKs6bpbONnP+cK/xe1LH8HtC3+l1LYprJu7EmETBCEUSAV8MUVotMAlYBQ4DtSIoij/fKfwQKDY+t99zLbkKpWKgICAe7059w3HR+H7VVBj471Po4KP7ITPJZvs+jfKtdZjXPr2C1xr6JSNs48Iwu3vPozu8ZybNhRRzqsJURTp7e3lzTfftNk/zGAwoFarJVEtWy/uHh4e5OXlER5u+qp98uRJiouLGR0dtUyj0+lIS0tjbm6Ouro6SSQvMjKS3NxcPD09mZycpK6ujtbWVomAcnV1JS0tjbi4OJmr4pkzZzh37hzd3d2MjNz45iUIAjt27CAnJwcPDw/JPGahVl9fb7MptbOzM4WFhYSFhXHixAlqa2ut9odbrRE3mI53f38/TU1NHD9+XHIM1Wo1wcHBhIeHExoaumpUztayr1y5QmNjI8ePH5f1WfPw8CA+Pp6YmBiuXr3Kvn37ZM6YZjQaDUaj8aZaBSzH0dFRkrZqxizczNjZ2VmEm7X9Vu7Xh5OV57VmEL5ZCqet9Ha3U8Hnk01tADQ2sjTm+wYY+fQPmGuVRrsdYiPweuEb2IdsuwN7obCSe3m/3tcpkYIgpAJPAYXAWiX8i0Ar8Hvgt6Iort45U+G+QhFsCveatQrGAfYGwzPpEHgTxnALg8OM/+BFpl8tlo2z3xGM299/HN0jGQiKccEtc+bMGQ4cOCCrPTJjZ2eHTqeTCDVbuLq6kpOTQ3R0NCqVivPnz1NUVCRxldTr9aSlpWEwGCgpKZEsNzg4mN27d+Pr68vMzAzl5eW0tLRIBIO3tzfp6ens2LFDZlwhiiInTpygrKxMYoKhVquJiYkhLS1N1ux6cXGRuro6qqqqbAo1FxcXiwDt7Oykrq5OEiVcTyNugLm5OTo7O2lqapKIVzBFIxMTE4mIiLA0q16Lubk5zp49y+joKGNjY5b/rowMWhPWDg4OpKamkpKSwpUrV3j99ddtpkoaDAZmZmaspruuF5VKhV6vt1oHuLIP3EoHUYW3H3OL8Hwj/FszLFn5jYn2MkXbQj3k4wDExUUuP/c7Lv/Tb2DxxnUraO1x//oncXnqvYpz8EPMfSnYBEH4EPAVYAewPO9iGlNUbRyYBdyv//MEzL9y4vVxfwS+J4ri2l7MChIEQfADvg3kAZsxpTBWAz8QRfH4avPewjoVwaZwT5hbhP/uMJmK2LJkjvaCb2VB8k30MV0an+DKL/7AxK9fkfVSU3t74P71T2J4f6HyQ3sbGB8f5/XXX7eZFqdWq3F2dpYYXtiKqBkMBrKysoiNjUWtVjM+Pk5paanEVdLBwYHMzEyCgoIoLi7m9OnTlnFeXl4UFhYSGBhoMRMpLy+XvMQHBASQnp5OcHCw1RTD4eFhjhw5wpkzN2pY7O3tSUhIICUlRZb6aLb0f+utt2xGFb28vNi9ezdbt26lubmZpqYmSdqig4ODZfm2GnEDjI6O0tTUREdHh0RMqdVqoqKiSExMZMuW9d0wk5OT9Pb20tvby5kzZ245+qXVasnIyCAxMZELFy5w8OBBmZg04+XlxdjY2C2tU6vVotfrZc6ZgiDg6Ogoqd0z1wgmJSXJeuApvD3oGIYvFcFJK9+T7NXwnWyTk6StrOO5jl4uPv19Fk5IW5Fok6LZ9PzXsA+20fRN4YHmvhJsgiDkAP8ExGISauPAn4FKoEEUxT4b8+mBBCAZeBxT6iTANeA54EeiKFr/9VKQIAjCJqALsAf+DTgDBAGfwlSvuEsUxdO2l3DT61UEm8JdRRRN9QM/rDbZ9Vtjs95kw/zOMFNNwkYwTl3lyq/+j4l/exnj1Io6Hictrp/+AK5P/yUqvVLfcqvMzc1x8OBBibHHclQqFZs2bWJkZGTNmiUnJycyMzOJj4/Hzs6Oq1evUlVVRVNTk+WlXq1Wk5iYSFJSEo2NjRLHSa1Wy+7du4mPj0elUtHX18fhw4clL/OBgYEW0WSNqakpSktLaW9vtwzTarWkpaWRmJhoNUIzPDzMvn37bEaU/P39ycvLw9XVlZqaGtra2iTRN2dnZ1JSUoiLi7PZFNrcBqCpqUlS7wc3XBLj4uLWVbM1OjrKsWPH6O3t5cIFuTMqmFJMN23ahKenJ56enmzatAmNRsPo6Cijo6OMjIwwOjoqMwMx4+joSFZWFgkJCQwPD/PGG2/YFG5OTk5WTVE2gsFgQK/Xy/bHmnBb
```

Comment:
here we can add something like:
"The small shift of the two curves reflects the shift of the peak energy between the  single delta function model and the full DT model"